src/ResourceManager/Sql/Commands.Sql/Common/AzureSqlCmdletBase.cs

```diff
@@ -59,87 +36,5 @@ protected virtual string GetResourceId(M model)
             HelpMessage = "The name of the resource group.")]
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
```

Comment:
@cormacpayne thanks for correcting. I wrote that comment and then a day later realized that is not right what I wrote, but by that time I was not able to come back to GitHub to correct it.